src/MoBi.UI/Views/CreateModuleView.cs

```diff
@@ -36,6 +36,7 @@ public override void InitializeResources()
          cbMoleculeStartValues.Text = MoleculeStartValues;
          cbParameterStartValues.Text = ParameterStartValues;
          createBuildingBlocksGroup.Text = CreateBuildingBlocks;
+         tbModuleName.Select();
```

Comment:
Making sure the name is focused and has the cursor when the view is shown

Reply:
I think there is a method to override in the view. This sometime does not work
 protected virtual void SetActiveControl()
      {
      }

```
  protected override void SetActiveControl()
      {
         ActiveControl = tbName;
      }
```